### ISSUE TITLE
HPack 0.2.0 ?

### ISSUE THREAD

**Issue by FredAns** [sorpaas/HTTP2.jl#7]
Hi,

I'm starting a project where I would like to drive a C++ application from the Julia command line, through a grpc connection ( I've already developed the python/grpc prototype).

Installing gRpc.jl requires HTTP2 which requires HPack 0.2.0. but I can only find the HPack 0.1.0 release... Could you help me please :) ?

Thanks !
Fred

**Comment by PallHaraldsson** [comment]
TL;DR, it seems you want to do something similar to (I'm not sure how, exactly, and this may work for you; don't have time to debug why I get error):

julia> using Pkg; Pkg.add(PackageSpec(url="https://github.com/tanmaykm/HTTP2.jl/tree/tan/julia10", rev="julia10"))
   Cloning git-repo `https://github.com/tanmaykm/HTTP2.jl/tree/tan/julia10`
ERROR: failed to clone from https://github.com/tanmaykm/HTTP2.jl/tree/tan/julia10, error: GitError(Code:ERROR, Class:Net, unexpected HTTP status code: 404)
[..]

I do see https://github.com/sorpaas/HPack.jl/tree/v0.2.0 but it's three years old, so it seem you want to user new/er on master.

Note however you may want  to (eventually) use: https://github.com/JuliaWeb/HTTP.jl/issues/448

I'm not sure it's too helpful to point to that unless you want to help with it (and integrate gRPC.jl to it). There's possibly also e.g. C code (or the Python code you already have; usable through PyCall) similar to gRPC that already uses non-Julia HPACK, and you could use that instead of gRPC.jl.
